js/form.js

```diff
@@ -1,10 +1,10 @@
-const  formAdState = () => {
+const  initFormStartState = () => {
   const formAd = document.querySelector('.ad-form');
   formAd.classList.toggle('ad-form--disabled');
-  formAd.toggleAttribute('disabled');
+  formAd.toggleAttribute('disabled', true);
```

Comment:
Да, но теперь уже не toggle, а set
```suggestion
  formAd.setAttribute('disabled', true);
```